index.d.ts

```diff
@@ -3,6 +3,7 @@ declare module 'react-native-matomo-sdk' {
     export function initialize(apiUrl: string, siteId: number): Promise<void>;
     export function trackView(route: string[]): Promise<void>;
     export function trackEvent(category: string, action: string, name?: string, value?: number): Promise<void>;
+    export function setUserId(userId: string): Promise<void>;
```

Comment:
I think it should be possible to remove the userId as well. At least the Android SDK does this by passing a `null` value as the userId (https://github.com/matomo-org/matomo-sdk-android/blob/v4.1.2/tracker/src/main/java/org/matomo/sdk/Tracker.java#L332). Not sure how the iOS SDK handles it though.

Reply:
iOS SDK works the same: https://github.com/matomo-org/matomo-sdk-ios/blob/develop/MatomoTracker/MatomoTracker.swift#L22